### PR TITLE
[LDA-2832] fix(configs): default sdk info always overwrite options values

### PR DESCRIPTION
**Pull request by mt-max**
This does not affect SDK functionality, it is just not reporting correct analytic metrics.

### CODE REVIEW

**Comment by mt-dfrey** [comment]
I don't have access to [LINK-2832](https://moneytree-app.atlassian.net/browse/LINK-2832) 🤔
Screen Shot 2023-03-06 at 9 34 32

**Comment by mt-max** [comment]
Mistake, LDA instead of LINK

File: src/helper.ts [hunk 1]
```diff
@@ -107,9 +95,9 @@ export function generateConfigs(configs: ConfigsOptions = {}): string {
   }
 
   return stringify({
-    sdk_platform: 'js',
-    sdk_version: __VERSION__,
-    ...snakeCaseConfigs
```
Comment (mt-max) [inline src/helper.ts:112]: Previously when `snakeCaseConfigs` contains `sdk_*: undefined`, it overwrites the [Line 110](https://github.com/moneytree/mt-link-javascript-sdk/pull/124/files#diff-a09b7f67f5f992f7ef9d9299f6fac774fb218572c9c2960eb2c30293067a8843L110) and [111](https://github.com/moneytree/mt-link-javascript-sdk/pull/124/files#diff-a09b7f67f5f992f7ef9d9299f6fac774fb218572c9c2960eb2c30293067a8843L111), hence no SDK information was included.

**Comment by mt-max** [comment]
We will make a release after doing a final test.

**Pull request merged**